src/test/scala/samples/MyTest.scala

```diff
@@ -0,0 +1,68 @@
+package samples
+
+import org.junit._
+import org.junit.runner.RunWith
+import org.specs2.mutable._
+import org.specs2.runner._
+import Getsetapi.Example
+import java.util.concurrent.ConcurrentHashMap
+
+
+@RunWith(classOf[JUnitRunner])
+class GetNoneTest extends Specification {
```

Comment:
1. The class and file names should be the same.
2. GetNoneTest does not indicate what is being tested here.